index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+    <link rel="stylesheet" href="assets/css/style.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
+      </style>
+    <title>Document</title>
+    
+</head>
+<body>
+    <header class="navegation-menu">
+    <div class="humburguer"><i href="#" class="fa fa-bars"></i>
+    </div> 
+    </header>
+
+    <main>  
+        <section class="headline-section">
+
+
+            <div class="headline-container">
+                <div class="header-text">
+                    <h1>Hey there.</h1>
+                    <h1>I'm Graciano.</h1>
+                    <h2>I'm a software developer</h2>
+                </div>
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don't hesitate to contact me.
+                </p>
+          
+<ul class="socialmedia-links">
+    <li><a href="https://twitter.com/GracianoSoft"><img src="assets/img/twitter.svg" alt=""></a></li>
+    <li><a href="https://www.linkedin.com/in/graciano-manuel-henrique-175527221/"><img src="assets/img/Linkedin icon.svg" alt=""></a></li>
+    <li><a href="https://medium.com/@gracianomanuelhenrique"><img src="assets/img/medium.svg" alt=""></a></li>
+    <li><a href=" https://github.com/Graciano1997"><img src="assets/img/github.svg" alt=""></a></li>
+    <li><a href="#"><img src="assets/img/angellist.svg" alt=""></a></li>
+  
+</ul>
+            </div>
+            
+        </section>
+    </main>
+    <footer></footer>
```

Comment:
- [ ] Kindly consider removing the `footer` element since you're currently not using it.